Automattic/jetpack
# Consolidate excluded files VS Code settings

## Proposed changes:

This adds most of the common excludes from our VSC settings into `files.exclude`. Making this global should help as some plugins inherit from that, and of course it will help with VSC directly.

That said, not all plugins do in fact inherit it, so it's still necessary/good practice to define it for every plugin.

### Other information:


## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






- Open the monorepo from VSC.
- You shouldn't observe any obvious slowdowns and the same files should still be ignored for both Intelephense and PHPCodeSniffer.
